### ISSUE TITLE
Missing dependency declaration

### ISSUE THREAD

**Issue by SlimeDog** [InstantlyMoist/xRay#2]
Spigot 1.15.2 (Jenkins 2596)
xRay 3.01

**Description**
Beginning with Spigot 1.15.2 (Jenkins 2596), missing dependency declarations are reported in the log on server start-up as WARN messages.

Reference: [https://hub.spigotmc.org/jira/browse/SPIGOT-5546](https://hub.spigotmc.org/jira/browse/SPIGOT-5546)

The relevant log entry is:
```
[22:03:30] [Server thread/WARN]: [xRay] Loaded class com.comphenix.protocol.wrappers.WrappedBlockData from ProtocolLib v4.5.1-SNAPSHOT-b445 which is not a depend, softdepend or loadbefore of this plugin.
```

**Steps to reproduce the behavior**
- Update Spigot to 1.15.2 latest
- Start the server
- Search the log for 'Loaded class'

**Expected behavior**
Dependencies should be declared in plugin.yml

**Comment by InstantlyMoist** [comment]
Fixed in 3.011

**Comment by SlimeDog** [comment]
Spigot 1.15.2
xRay 3.011

WARN remains on server start-up
```
[08:02:25] [Server thread/WARN]: [xRay] Loaded class com.comphenix.protocol.wrappers.WrappedBlockData from ProtocolLib v4.5.1-SNAPSHOT-b445 which is not a depend, softdepend or loadbefore of this plugin.
```

**Comment by InstantlyMoist** [comment]
Will be checking this again, thank you.

**Comment by InstantlyMoist** [comment]
Fixed in version 3.012

**Comment by SlimeDog** [comment]
Verified fix in 3.012. Thank you.
